tags/subchildren/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "subchildren",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
+}
```

Comment:
I thought it was but I just found an example where it doesn't use the module structure. I'ved updated the PR.